public/sw.js

```diff
@@ -229,9 +228,37 @@ self.addEventListener('fetch', evt => {
 
 
 
+// Install a service worker
+// self.addEventListener('install', event => {
+//   //perform install steps
+//   event.waitUntil(
+//     caches.open(cacheName)
+//     .then(cache => {
+//       console.log('Opened cache')
+//       return cache.addAll(urlsToCache)
+//     })
+//   )
+// })
+
+// self.addEventListener('fetch', event => {
+//   event.respondWith(
+//     caches.open(cacheName).then(cache => {
+//       return cache.match(event.request).then(response => {
+//         return (
+//           response ||
+//           fetch(event.request).then(response => {
+//             cache.put(event.request, response.clone());
+//             return response;
+//           })
+//         );
+//       });
+//     }),
+//   );
+// });
+
+
```

Comment:
All of this should have been removed with the last PR

Reply:
I merged them on gh. I'm not sure why they didn't take.

Reply:
It seems fixed now thanks